db/migrate/20220212200446_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] Since likes belong to a user and a post, you should add references to both the user and to the post.